src/ShallowWrapper.js

```diff
@@ -3,6 +3,7 @@ import flatten from 'lodash/flatten';
 import unique from 'lodash/uniq';
 import compact from 'lodash/compact';
 import cheerio from 'cheerio';
+import merge from 'lodash/merge';
```

Comment:
I think we needs `merge` because `options` might be nested object.

```js
this.options = { context: {foo: 'hello'}};
options = {context: {bar: 'enzyme!'}};
assign({}, this.options, options)  // It will be {context: {bar: 'enzyme!'}}
```